### ISSUE TITLE
The use of the private `{{-in-element}}` is deprecated

### ISSUE THREAD

**Issue by onechiporenko** [kaliber5/ember-bootstrap#1172]
New project with only `ember-bootstrap` (latest 3.x) installed causes deprecations about "private `{{-in-element}}`" usage.

Full log with "STR" is below:

```bash
$ ember init && npm i && ember i ember-bootstrap@3 && npm i && ember server
installing app
Ember CLI v3.20.0

Creating a new Ember app in D:\git\bs3-test:
  create .editorconfig
  create .ember-cli
  create .eslintignore
  create .eslintrc.js
  create .template-lintrc.js
  create .travis.yml
  create .watchmanconfig
  create README.md
  create app\app.js
  create app\components\.gitkeep
  create app\controllers\.gitkeep
  create app\helpers\.gitkeep
  create app\index.html
  create app\models\.gitkeep
  create app\router.js
  create app\routes\.gitkeep
  create app\styles\app.css
  create app\templates\application.hbs
  create config\ember-cli-update.json
  create config\environment.js
  create config\optional-features.json
  create config\targets.js
  create ember-cli-build.js
  create .gitignore
  create package.json
  create public\robots.txt
  create testem.js
  create tests\helpers\.gitkeep
  create tests\index.html
  create tests\integration\.gitkeep
  create tests\test-helper.js
  create tests\unit\.gitkeep
  create vendor\.gitkeep

Installing packages... This might take a couple of minutes.
- npm: Installing dependencies ...
npm: Installed dependencies

Successfully created project bs3-test.
Get started by typing:

  $ cd bs3-test
  $ npm start

Happy coding!
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\watchpack-chokidar2\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

audited 2041 packages in 8.205s

48 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
- Installing addon package

- Installing addon package
Installing packages... This might take a couple of minutes.
- Installing addon package
- npm: Installing ember-bootstrap@3 ...
npm: Installed ember-bootstrap@3
installing ember-bootstrap
Installing for Bootstrap 4 using preprocessor none
  install package bootstrap

Installing packages... This might take a couple of minutes.
- npm: Installing bootstrap@^4.3.1 ...
npm: Installed bootstrap@^4.3.1
Added ember-bootstrap configuration to ember-cli-build.js
Installed addon package.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules\watchpack-chokidar2\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

audited 2156 packages in 7.56s

50 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details

Running without permission to symlink will degrade build performance.
See https://cli.emberjs.com/release/appendix/windows/ for details.

- building...
DEPRECATION: The use of the private `{{-in-element}}` is deprecated, please refactor to the public `{{in-element}}`. ('ember-popper/templates/components/ember-popper-targeting-parent.hbs' @ L1:C0)  [deprecation id: glimmer.private-in-element]
        at logDeprecationStackTrace (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1917:21)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at raiseOnDeprecation (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1944:9)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at invoke (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2026:9)
        at deprecate (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1982:28)
        at BlockStatement (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:10801:59)
        at visitNode (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7729:16)
        at visitArray (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7820:20)
DEPRECATION: The use of the private `{{-in-element}}` is deprecated, please refactor to the public `{{in-element}}`. ('ember-popper/templates/components/ember-popper.hbs' @ L1:C0)  [deprecation id: glimmer.private-in-element]
        at logDeprecationStackTrace (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1917:21)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at raiseOnDeprecation (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1944:9)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at invoke (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2026:9)
        at deprecate (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1982:28)
        at BlockStatement (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:10801:59)
        at visitNode (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7729:16)
        at visitArray (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7820:20)
DEPRECATION: The use of the private `{{-in-element}}` is deprecated, please refactor to the public `{{in-element}}`. ('ember-bootstrap/templates/components/bs-modal-simple.hbs' @ L1:C0)  [deprecation id: glimmer.private-in-element]
        at logDeprecationStackTrace (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1917:21)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at raiseOnDeprecation (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1944:9)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at invoke (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2026:9)
        at deprecate (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1982:28)
        at BlockStatement (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:10801:59)
        at visitNode (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7729:16)
        at visitArray (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7820:20)
DEPRECATION: The use of the private `{{-in-element}}` is deprecated, please refactor to the public `{{in-element}}`. ('ember-bootstrap/templates/components/bs-modal.hbs' @ L1:C0)  [deprecation id: glimmer.private-in-element]
        at logDeprecationStackTrace (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1917:21)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at raiseOnDeprecation (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1944:9)
        at HANDLERS.<computed> (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2014:9)
        at invoke (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:2026:9)
        at deprecate (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:1982:28)
        at BlockStatement (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:10801:59)
        at visitNode (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7729:16)
        at visitArray (D:\git\bs3-test\node_modules\ember-source\dist\ember-template-compiler.js:7820:20)

Build successful (10640ms) – Serving on http://localhost:4200/

```

Output from `ember bootstrap:info`:
```
$ ember bootstrap:info
Npm packages:
ember-bootstrap: ^3.1.4 -> 3.1.4
ember-cli: ~3.20.0 -> 3.20.0
bootstrap: ^4.5.0 -> 4.5.0
bootstrap-sass: n/a
ember-cli-sass: n/a
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapCSS: true
```

**Comment by onechiporenko** [comment]
Should `ember-popper` be updated to `v0.11.2` in the [package.json](https://github.com/kaliber5/ember-bootstrap/blob/v3.1.4/package.json#L106)?

**Comment by jelhan** [comment]
This is a duplicate of #1162. It's fixed in latest v4 release candidate. I don't think we have bandwidth to backport the fix. Especially if as it's only a deprecation. But I think we would accept a pull request if someone is willing to invest the time.

**Comment by jelhan** [comment]
@simonihmig Just released v4.0.0: https://github.com/kaliber5/ember-bootstrap/releases/tag/v4.0.0 Upgrading to this new stable version will fix your issue.

**Comment by onechiporenko** [comment]
I see `v4` now is stable release, so I'm closing this issue.
